moviepy/video/io/ffmpeg_reader.py

```diff
@@ -435,7 +436,7 @@ def parse(self):
                 stream_type_lower = stream_type.lower()
 
                 if language is not None:
-                    if language == "und" or language.startswith("0x"):
+                    if language == "und":
```

Comment:
You can rewrite this just as `if language == "und"`

Reply:
You're right, thanks!